language-scripts/nodejs/dev-run

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+set -x
+
+cd /tmp/src
+
+
+NPM_RUN="${NPM_RUN:=start}"
+NODE_ENV="${NODE_ENV:=development}"
+
+
+export GIT_COMMITTER_NAME="unknown"
+export and GIT_COMMITTER_EMAIL="unknown@localhost.com"
```

Comment:
I'm assuming this is needed? Could we set this "odo specific" or no? For example odo@localhost or odo for git committer name?

Reply:
It is a workaround for some git issue http://git.661346.n2.nabble.com/git-clone-fails-when-current-user-is-not-in-etc-passwd-td7643604.html
I just took it from original nodejs assemble script.
It is only to make sure that git is not complaining about the nonexisting user when doing `git clone` otherwise it is not used anywhere.

Reply:
Minor: suggest "unknown@localhost" i.e. drop the .com. "localhost" won't trigger DNS lookups or other funny business. "localhost.com" sounds like a phishing site you shouldn't open emails from :)